### PR TITLE
add configurable per state stats

### PR DESCRIPTION
**Pull request by krystiancharubin**

### CODE REVIEW

**Comment by krystiancharubin** [comment]
#25 - can just leave stats empty to remove
#12 - can choose what to render

**Comment by insertjokehere** [comment]
This idea looks good - I use an Ecovacs vacuum, which uses a different format for its component stats attributes (`component_<name>` rather than `<name>_left`)

File: README.md [hunk 1]
```diff
@@ -86,8 +107,17 @@ Here is what every option means:
 | `image`        | `string`  | `default`    | Path to image of your vacuum cleaner. Better to have `png` or `svg`.    |
 | `show_name`    | `boolean` | `true`       | Show friendly name of the vacuum.                                       |
 | `show_toolbar` | `boolean` | `true`       | Show toolbar with actions.                                              |
+| `stats`        | `object`  | Optional     | Custom per state stats for your bacuum cleaner                          |
```
Comment (pkishino) [inline README.md:110]: Bacuum??

File: README.md [hunk 2]
```diff
@@ -86,8 +107,17 @@ Here is what every option means:
...
 | `actions`      | `object`  | Optional     | Custom actions for your vacuum cleaner.                                 |
 
+### `stats` object
+
+| Name        |   Type   | Default  | Description                                     |
+| ----------- | :------: | -------- | ----------------------------------------------- |
+| `attribute` | `string` | Optional | Attribute name of the stat, i.e. `filter_left`. |
+| `unit`      | `string` | Optional | Unit of measure, i.e. `hours`.                  |
+| `subtitle`  | `string` | Optional | Firendly name of the stat, i.e. `Filter`.       |
```
Comment (pkishino) [inline README.md:119]: Firendly => Friendly

**Pull request merged**